src/Tribe/Template.php

```diff
@@ -321,9 +321,9 @@ protected function get_template_plugin_path() {
 	 * @return array
 	 */
 	protected function get_template_public_namespace() {
-		$namespace = array(
+		$namespace = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: b2d77e8f1cd21a0066f3c8a59733d83d</sub>